### PR TITLE
Fix return issue with chain/yield

### PR DESCRIPTION
**Pull request by bluemoon**
@sameerdp5 

Also the diff is very confusing

### CODE REVIEW

File: normalize/diff.py [hunk 1]
```diff
@@ -675,110 +675,110 @@ def compare_collection_iter(propval_a, propval_b, fs_a=None, fs_b=None,
         for key in shared_keys:
             if (isinstance(propval_a, collections.Iterable) and
                isinstance(propval_b, collections.Iterable)):
-
                 diffs = _diff_iter(propval_a[key], propval_b[key],
                                    fs_a + [key], fs_b + [key], options)
+
                 for diff in diffs:
                     yield diff
             # early exit
```
Comment (sameerdp5) [inline normalize/diff.py]: can remove comments

**Comment by sameerdp5** [comment]
bump version!

**Comment by sameerdp5** [comment]
lgtm

**Comment by adepue** [comment]
lgtm  @sameerdp5   ?w=1 is your friend

**Comment by sameerdp5** [comment]
merge it!

**Pull request merged**